### PR TITLE
Spice up the FAQ

### PR DESCRIPTION
**Pull request by Tzky**
This PR:

* adds a nice and shiny table at the top of the FAQ
*  changes some Q's for better readability

As always comments and thoughts are more than welcome :)

This PR was created to allow for preview, I closed #313 in favor of this one. All comments made there should already be adressed in this (new) PR.

### CODE REVIEW

**Comment by Spongy** [comment]
Setting up PR reference, please wait...

File: source/about/faq.rst [hunk 1]
```diff
@@ -119,16 +116,15 @@ While this does not free us from Mojang’s control, as their code is proprietar
...
 which has a large number of names de-obfuscated. However, be aware that accessing the server internals raises the risk
 of your plugin breaking - this is your prerogative.
 
 Will the Sponge API Be Similar to the Bukkit API in Terms of Usability?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-----------------------------------------------------------------------
 
 Yes. It will also afford you more power because you will have access to Forge.
-
 It is worth noting that the Spout API is serving as inspiration for the development of the Sponge API, which reduces the
```
Comment (Aaron1011) [inline source/about/faq.rst]: While this isn't added in this PR, it should probably be removed. While individual developers may draw their inspiration from various projects, this is (now) much too broad of a statement to be meaningful.

Comment (Tzky) [inline source/about/faq.rst]: I'm not happy with this Q either. removed it for now.

**Comment by Tzky** [comment]
Hmm, i think we're good to go. 
There shouldn't be any flaws left in here, hopefully.

Any further thoughts?

**Comment by RobertHerhold** [comment]
Thanks! :)

**Pull request merged**